CONTRIBUTING.md

```diff
@@ -1,3 +1,11 @@
 # Contributing guide
 
-This section is under construction.
\ No newline at end of file
+This section is under construction.
+
+## Troubleshooting
+
+**`cargo build` fails because cannot build `openssl-sys`**
+
+This is due to missing packages in your system, install `libssl-dev` to fix this.
+
+- Ubuntu: `sudo apt update && sudo apt install libssl-dev protobuf-compiler`
```

Comment:
Will be modified in an other PR ?

Reply:
Yeah

Reply:
Can you create an issue for it pls ? @thomasgouveia

Reply:
Done here https://github.com/rik-org/rik/issues/40